baldur/README.md

```diff
@@ -10,6 +10,9 @@ The following configuration files must be available in the /config directory.
 
 The configuration in baldur.properties serves two purposes:  to allow certain users access to the service and to describe the permission rules for URIs that baldur will produce through its REST API.
 ```
+# time (in seconds) the grant is considered valid 
```

Comment:
For consistency with other service config, the property keys should be 
```
org.opencadc.baldur.grantExpiry = {time in seconds}
org.opencadc.baldur.allowedUser = {user identity}
```
and specify that multiple occurances of allowedUser is supported. For consistency with other places (eg. logControl) where we configure usage permissions like this, I would also add:
```
org.opencadc.baldur.allowedGroup = {groupURI}
```
also multiple occurances. Then add "TODO: add support for allowedGroup" and create an issue so it's clear that it isn't supported but this is how it will be eventually. (see raven README and writeGrantProvider for how it is expressed there).

Instead of entry:
```
org.opencadc.baldur.entry = {entry name} {regular expression}
```

Reply:
Instead of a single line for user or groups with space delimited entries, multiple lines for each. Okay that makes it easier to read.
Was there something more about ...baldur.entry ?

Reply:
Comment updated to clarify `entry` -> `org.opencadc.baldur.entry`

Reply:
Renamed the property keys and created an issue to add support for allowdGroup.